loom/engine/loom2d/l2dDisplayObject.h

```diff
@@ -310,6 +316,46 @@ class DisplayObject : public EventDispatcher
      *  instead of creating a new object. */
     void getTargetTransformationMatrix(DisplayObject *targetSpace, Matrix *resultMatrix);
 
+
+    static void transformBounds(Matrix *transform, Rectangle *bounds, Rectangle *resultRect) {
+        lmAssert(transform != NULL, "Transform is null");
+        lmAssert(bounds != NULL, "Bounds are null");
+        lmAssert(resultRect != NULL, "Result rect is null");
+
+        float rx, ry;
+        float minx, miny, maxx, maxy;
+
+        transform->transformCoordInternal(bounds->getLeft(), bounds->getTop(), &rx, &ry);
```

Comment:
Perhaps, thought the above is how Starling does it. Maybe we can just do it the old way if a & b equal 0?